### DISCUSSION TITLE
Issue while starting an account on Azure with MLSA ID

### DISCUSSION THREAD

**Discussion by qaidjoharj53** [microsoft/SATechnicalOnboarding#102] [Questions]
Can I start technical onboarding after achieving Alpha milestone? So to get azure account on mlsa id?
![Screenshot 2024-01-24 174702](https://github.com/microsoft/SATechnicalOnboarding/assets/113633954/db182241-bc96-4b58-a644-0e8ddee39bf4)

**Comment by majafrankowska** [comment]
Hey, you need to sign up using your University email to pass Verification process.

**Comment by AnthonyByansi** [comment]
Hey @qaidjoharj53 I would recommend you to start on your technical after attaining the alpha milestone, as that will give you access to $150 Azure monthly credits. Which can be accessed [Here](https://my.visualstudio.com/). It's possible to redeem the students credits, which might take a bit long. So advance to Alpha first.

![Azure Credits](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/c0069db6-7b02-4905-93e3-21dd1fbeb22c)

**Comment by NandiniAg05** [comment]
I'm not able to sign up in my azure account. I want to redeem my credits, but it is asking me to fill a form and even after filling, it showing nothing. Please help me out with this.
